### PR TITLE
[doc]ngql-bnf

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Changes in this pr:
1. update the nGQL format in BNF
2. Add `GO N STEPS`
2. Comment out the Prometheus doc because the code is under changes @Shylock-Hg 
4. Add java client in doc

### CODE REVIEW

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # INSERT VERTEX 语法
 
 ```ngql
-INSERT VERTEX tag_name[, tag_name] (prop_name_list[, prop_name_list])
-     {VALUES | VALUE} vid: (prop_value_list[, prop_value_list])
+INSERT VERTEX <tag_name> [, <tag_name>, ...] (prop_name_list[, prop_name_list])
+     VALUES | VALUE} vid: (prop_value_list[, prop_value_list])
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md]: INSERT VERTEX <tag_name> [, <tag_name>, ...] (prop_name_list[, prop_name_list, ...])
VALUES | VALUE} <vid>: (prop_value_list [, prop_value_list, ...])

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 prop_name_list:
-  [prop_name [, prop_name] ...]
+  [prop_name [, prop_name] ...
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md]: why delete ]

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-syntax.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # SHOW 语法
 
 ```ngql
-SHOW {SPACES | TAGS | EDGES | HOSTS | PARTS}
+SHOW SPACES | TAGS | EDGES | HOSTS | PARTS
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-syntax.md]: SHOW SPACES { TAGS | EDGES | HOSTS | PARTS }

see [mysql manual](https://dev.mysql.com/doc/refman/8.0/en/show.html)

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md (hunk 1 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md]: vid is a must here

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/alter-tag-edge-syntax.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # 修改 Tag / Edge
 
 ```ngql
-ALTER {TAG | EDGE} tag_name | edge_name
-    [alter_definition [, alter_definition] ...]
+ALTER TAG | EDGE <tag_name> | <edge_name>
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/alter-tag-edge-syntax.md]: ALTER { TAG <tag_name> | EDGE  <edge_name> }

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
 The `DELETE EDGE` statement is used to delete edges. Given an edge type, the source vertex and the dest vertex, **Nebula Graph** supports `DELETE` the edge, its associated properties and the edge ranking. You can also delete an edge with a certain rank. The syntax is as the follows:
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md]: as follows

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/update-vertex-edge-syntax.md [hunk 6]
```diff
@@ -5,17 +5,17 @@
 ## Update vertex
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/update-vertex-edge-syntax.md]: Uppercase

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-edge-syntax.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
 # INSERT EDGE 语法
 
 ```ngql
-INSERT EDGE edge_name ( <prop_name_list> ) {VALUES | VALUE}
+INSERT EDGE <edge_name> ( <prop_name_list> ) VALUES | VALUE
 <src_vid> -> <dst_vid>@ranking : ( <prop_value_list> )
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-edge-syntax.md]: I think `<ranking>` is better

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md [hunk 8]
```diff
@@ -8,10 +8,10 @@ It indicates to travel in a graph with specific filters (the `WHERE` clause), to
...
-  OVER <edge_type_list>
-  WHERE (expression [ AND | OR expression ...])  
-  YIELD | YIELDS  [DISTINCT] <return_list>
+  GO [ <N> STEPS ] FROM <node_list>
+  OVER <edge_type_list> [REVERSELY]
+  [ WHERE <expression> [ AND | OR expression ...]) ]
+  YIELD | YIELDS [DISTINCT] <return_list>
 
 <node_list>
    | vid [, vid ...]
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md]:
```
<vid> is better
```

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md [hunk 9]
```diff
@@ -1,9 +1,9 @@
 # Delete Edge Syntax
 
-The `DELETE EDGE` statement is used to delete edges. Given an edge type, the source vertex and the dest vertex, **Nebula Graph** supports `DELETE` the edge, its associated properties and the edge ranking. You can also delete an edge with a certain rank. The syntax is as the follows:
+The `DELETE EDGE` statement is used to delete edges. Given an edge type, the source vertex and the dest vertex, **Nebula Graph** supports `DELETE` the edge, its associated properties and the edge ranking. You can also delete an edge with a certain rank. The syntax is as follows:
 
 ```ngql
-DELETE EDGE <edge_type> <vid> -> <vid>@ranking [, <vid> -> <vid> ...]
+DELETE EDGE <edge_type> <vid> -> <vid>[@ranking] [, <vid> -> <vid> ...]
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md]: `<ranking>`

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-edge-syntax.md (hunk 7 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-edge-syntax.md]: But the syntax requires a @, so I leave it as it is.

Comment (whitewum) [reply docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-edge-syntax.md]:
```
<src_vid> -> <dst_vid>[@<ranking>] : ( <prop_value_list> )
```
but it's ok. fix next time

**Pull request merged**